### PR TITLE
Alert component added

### LINKED ISSUE #17: Alert component required
In NitroHelp we are using the component `SweetAlert` which is in shared folder. 
Alert Component is required in nitroform and nitroacademy. 

cc: @vinay0x

### PR DESCRIPTION
**Pull request by edwinbbu**
Fixes #17

Screenshot 2020-10-29 at 2 43 04 PM

I have tested the component in nitrohelp.
I have hardcoded 'Are you sure?' in order to have Consistency vocabulary as mentioned by Neeraj.

@vinay0x _a please review

### CODE REVIEW

File: lib/components/Alert.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      cancelButtonProps={{
+        style: "secondary",
+        label: cancelActionText || "Cancel",
+        onClick: cancelAction
+      }}
+      onClose={cancelAction}
+    >
+      <div className="flex">
+        <div className="flex items-center justify-center flex-shrink-0 w-10 h-10 bg-red-100 rounded-full">
+          <i className="text-red-500 ri-alert-line ri-lg"></i>
```
Comment (vinay0x) [inline lib/components/Alert.js:25]: Can you make this a prop? Show the alert icon as default though.

File: lib/components/Alert.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+        <div className="ml-4">
+          <h3 className="mb-2 text-lg font-medium text-gray-700">
+            {title || "Delete"}
+          </h3>
+          <div className="text-sm leading-5 text-gray-500 mb-2">
+            {message}
+          </div>
+          <div className="text-sm leading-5 text-gray-500">
+            Are you sure?
```
Comment (vinay0x) [inline lib/components/Alert.js:36]:
```suggestion
            Are you sure you want to continue?
```
Doesn't this sound bettter?

File: lib/components/Alert.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      onClose={cancelAction}
+    >
+      <div className="flex">
+        <div className="flex items-center justify-center flex-shrink-0 w-10 h-10 bg-red-100 rounded-full">
+          <i className="text-red-500 ri-alert-line ri-lg"></i>
+        </div>
+
+        <div className="ml-4">
+          <h3 className="mb-2 text-lg font-medium text-gray-700">
+            {title || "Delete"}
```
Comment (vinay0x) [inline lib/components/Alert.js:30]:
```suggestion
            {title}
```

**Review by vinay0x — CHANGES_REQUESTED** [review]
@edwinbbu _a

File: lib/components/Alert.js (hunk 2 above)
Comment (edwinbbu) [reply lib/components/Alert.js]: Yes. I have added it

File: lib/components/Alert.js (hunk 1 above)
Comment (edwinbbu) [reply lib/components/Alert.js]: Fixed

File: lib/components/Alert.js (hunk 3 above)
Comment (edwinbbu) [reply lib/components/Alert.js]: Done

File: lib/components/Alert.js (hunk 2 above)
Comment (edwinbbu) [reply lib/components/Alert.js]: Screenshot 2020-10-30 at 9 59 21 AM

**Comment by edwinbbu** [comment]
@vinay0x _a I have updated the requested changes. Please review

**Pull request merged**